app/javascript/packs/completed-needs.js

```diff
@@ -0,0 +1,10 @@
+document.querySelectorAll('.completed-need').forEach(e => e.style.display = 'none');
```

Comment:
Didn't Jamie add collapsing code for assessments already? Perhaps it would make sense to use that since it's the same functionality?

Reply:
Good point, thanks. Not sure will have time to collect the logic from two spaces in one in next 30 mins.